posts/react-compound-components/2021-5-25-React-compound-compnents.md

```diff
@@ -0,0 +1,250 @@
+---
+templateKey: "blog-post"
+title: "React compound components (Hooks + typescript)"
+date: 2021-05-25
+featuredpost: false
+description: >-
+  This article talks about what are compound components and how to build an accordion component using the concept of compound components.
+keywords:
+- reactjs
+- Compound-components
+- Hooks
+- Typescript
```

Comment:
tutorial
react context

Reply:
Do i need to append these two or replace these two with the other keywords?

Reply:
append